example/src/main/java/com/ably/chat/example/MainActivity.kt

```diff
@@ -46,20 +46,23 @@ val randomClientId = UUID.randomUUID().toString()
 class MainActivity : ComponentActivity() {
```

Comment:
I do not support pure kotlin code for UI, instead it should be put into layout.xml files.
This is more like writing website using pure js without use of html + css. 
One of the biggest drawback is lack of local preview when adjusting UI components. Right now, we need to rerun app to look for breaking UI components. Also, you can't preview layout on different devices.
I am assuming this is a temporary code and will need complete rewrite in the near future ( Since code is not clean and is a mix of UI + business code )

Reply:
No more xml in Android, it's old news :) with Jetpack Compose everything is in Kotlin, there are special preview component for UI. I added them.

Reply:
Wow, I didn't know so many things changed. I loved working with xml. Lots of threads talking about the same
https://www.reddit.com/r/androiddev/comments/1bbbsop/why_are_people_against_xml_now/

Reply:
Okay, I will check new way to develop UI layouts, thanks!

Reply:
https://medium.com/@MeenoTeK/xml-views-or-jetpack-compose-which-is-the-best-option-for-your-next-project-ccf38573a82